app/views/propertyrepresentation/appoint/agentToManageOneProperty.scala.html

```diff
@@ -0,0 +1,57 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import actions.requests.BasicAuthenticatedRequest
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@import config.ApplicationConfig
+@import models.propertyrepresentation.ManageOnePropertyOptions
+
+@import models.propertyrepresentation.AsWellAsCurrent
+@import models.propertyrepresentation.InsteadOfCurrent
+@import models.propertyrepresentation.No
+
+@this(govukErrorSummary : GovukErrorSummary, govukRadios : GovukRadios, govukButton : GovukButton, mainLayout: views.html.mainLayout)
+
+@(form: Form[ManageOnePropertyOptions], agentName: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+
+@mainLayout(
+    pageTitle = messages("propertyRepresentation.addAnAgent.title"),
+    backUri = Some(controllers.agentAppointment.routes.AgentRelationshipController.isCorrectAgent().url)
+) {
+
+    <h1 class="govuk-heading-xl">@messages("propertyRepresentation.oneProperty.question", agentName)</h1>
```

Comment:
can we discuss the differences between these screens?

Reply:
happy to discuss, come over when you are free